src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,115 @@
+package duke.storage;
+
```

Comment:
I think you might have missed out on the additional blank line here between your `package`  statement and `import` statements.